packages/calypso-e2e/src/element-helper.ts

```diff
@@ -1,16 +1,20 @@
 import { Locator, Page, Frame } from 'playwright';
 import envVariables from './env-variables';
 
-const navTabParent = 'div.stats-navigation__tabs';
-const legacyNavTabParent = 'div.section-nav';
+const coreNavTabParent = 'div.stats-navigation__tabs';
```

Comment:
feels like this is too specific... Now I doubt that it was a good idea to replace the tabs only for Stats...